contracts/protocol/facets/ConfigHandlerFacet.sol

```diff
@@ -226,14 +230,62 @@ contract ConfigHandlerFacet is IBosonConfigHandler, ProtocolBase {
     }
 
     /**
-     * @notice Gets the protocol fee percentage.
+     * @notice Sets the feeTable for a specific token given price ranges and fee tiers for
+     * the corresponding price ranges.
      *
-     * @return the protocol fee percentage
+     * Reverts if the number of fee percentages does not match the number of price ranges.
+     * Reverts if token is Zero address.
```

Comment:
This revert reason does not apply anymore.

Also, if some of the invoked functions revert, we also add that info in the natspec of the top-level function. So in this case, you should add that it reverts if ranges are not ascending or some of the percentages are over 100%.

The same comment applies to the natspec in `IBosonConfigHandler`